### ISSUE TITLE
LYCHEE 3.2.16 large icon index bug

### ISSUE THREAD

**Issue by quaiman** [LycheeOrg/Lychee-v3#271]
### Detailed description of the problem
Hello
I don't understand, I did follow all your installation tips but I can't have anything else than the bug page with all the big icons as you tell in the first FAQ topic : Lychee is not working

### Steps to reproduce the issue
I already have all my permissions set like you ask 755 except data/ with 750...
### Output of the diagnostics (Settings => Diagnostics)
Here is the output for Lychee/plugins/Diagnostics/index.php

Diagnostics
-----------
Warning: Dropbox import not working. No property for dropboxKey.
Warning: Pictures that are rotated lose their metadata! Please install Imagick to avoid that.
Error: 'data/config.php' is world readable.
Error: Username empty or not set in database
Error: Password empty or not set in database

System Information
------------------
You have to be logged in to see more information.

### Browser and system

I am working on a Mac Pro with MAC OS X 10.11.6 
and tested with Firefox 67.0.4 (64bits) and also with Safari 11.1.2 (11605.3.8.1)

here is a screenshot of the index page of my Lychee 3.2.16 index page :
![image](https://user-images.githubusercontent.com/2340599/60659485-cbcb0700-9e55-11e9-80d9-1e1e396283d8.png)

For your information, the previous version Lychee 3.1 was working perfectly on the same web-server...

Thank you for your help.

**Comment by ildyria** [comment]
I edited and toned down the aggressiveness/frustration of your issue.

Did you check that part?

> If you choose not to use HTTPS, you may need to disable or review the Content-Security-Policy header in Lychee's .htaccess file.

Basically, did you try to comment this line in the `.htaccess` :
> 	Header set Content-Security-Policy "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self'; connect-src 'self' https://lycheeorg.github.io; form-action 'none'; base-uri 'none'; frame-ancestors 'none'; upgrade-insecure-requests; block-all-mixed-content;"

**Comment by bennettscience** [comment]
Disabling the CSP line will fix the issue.

**Comment by quaiman** [comment]
ohhh sorry for the "infex" ... obviously a typo, 
thanks for intention of my indextion  :-)

About my problem. Thanks for your rapid answers guys,
BUT after added a # on the ligne "Header set Content-Security-Policy" in the .htaccess file
same thing, it was still bugging... same large icon index page...

Then, I try just to connect to the same Lychee folder but using HTTPS protocol, has ildyria tell me,

AND THEN, after a long and dark tunnel, I SAW THE LIGHT, a very bright a warm light, containing all my friends and all my dead family members and living Lychees !!! 

so long story short, it is working now using HTTPS and commented  the CSP line in Lychee .htaccess file

I'll be back to you after some test, and tell you my other wonderfull adventure in the lychee land...
Thanks

**Comment by ildyria** [comment]
Thus fixed.
